lib/fallback-middleware.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+// Attribution: https://github.com/bripkens/connect-history-api-fallback
```

Comment:
why don't you just send a PR to the original project and put this option behind a flag if they don't want to accept a backwards incompatible change (but maybe they do!).

in the meantime you can depend on your branch in the fork via package.json to get this issue fixed in lite-server now in case it takes a while for the PR to be merged.